Doc/library/inspect.rst

```diff
@@ -242,6 +242,12 @@ attributes:
 
    Add ``cr_origin`` attribute to coroutines.
 
+.. function:: getabsfile(object[, _filename])
+
+   Return an absolute path to the source or compiled file for an object.
+   The idea is for each object to have a unique origin, so this routine
```

Comment:
IMO this phrase could be remove because this sound like an "idea" and not like something concrete I do not know if I'm clear)

Reply:
@eamanu This is part of getabsfile docstring.